### DISCUSSION TITLE
Allow pasting tables greater than 6 columns

### DISCUSSION THREAD

**Discussion by Heavenira** [DesModder/DesModder#848] [Ideas]
Very frustrating limitation when importing CSVs. If you try pasting this into Desmos:
```js
1,2,3,4,5,6,7,8,9
1,2,3,4,5,6,7,8,9
```
only columns 1-6 will make it through. This is frustrating, as it forces the user to separate their data every 6 columns.

**Comment by jared-hughes** [comment]
Related: #241
